src/commands/gate/interfaces.ts

```diff
@@ -15,22 +15,17 @@ export interface EvaluationResponsePayload {
   }
 }
 
-type GateStatus = 'passed' | 'failed' | 'empty'
```

Comment:
We can introduce new states in future on the service side, and I don't want it to break backward compatibility with the command